<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransactionDialog</class>
 <widget class="QDialog" name="TransactionDialog">
  <property name="windowTitle"><string>Transaction</string></property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QComboBox" name="comboType">
     <item><property name="text"><string>expense</string></property></item>
     <item><property name="text"><string>income</string></property></item>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboCategory"/>
   </item>
   <item>
    <widget class="QDateEdit" name="dateEdit">
     <property name="date"><string>2025-01-01</string></property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="editAmount" >
     <property name="placeholderText"><string>Amount</string></property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="editNote" >
     <property name="placeholderText"><string>Note</string></property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item><widget class="QPushButton" name="btnOk"><property name="text"><string>OK</string></property></widget></item>
     <item><widget class="QPushButton" name="btnCancel"><property name="text"><string>Cancel</string></property></widget></item>
    </layout>
   </item>
  </layout>
 </widget>
</ui>
